src/classes/BDI_MatchDonations.cls

```diff
@@ -101,7 +101,11 @@ public with sharing class BDI_MatchDonations implements BDI_IMatchDonations {
 
         // now look for the matching Opp for each donation
         for (DataImport__c di : dataImports) {
-            if (di.DonationImportStatus__c != label.bdiMatchedByUserNewOpp) {
+            // User told us in BGE interface to not match this record and only create new Opportunity
+            // OR User told us in BGE interface to apply a new payment to a given Opportunity
+            // TODO: could this be summarized as "new opp OR donation already specified"?
```

Comment:
Is this a todo for a comment, or a standing work todo?

Reply:
Hey @bethbrains!  Could you weigh in here?  Thanks!